pmm-app/tests/verifyPMMSettingsPageElements_test.js

```diff
@@ -1,15 +1,16 @@
 const assert = require('assert');
 const page = require('./pages/pmmSettingsPage');
 
-const validationValues = ['text ', '2147483648', '-1', '<script>alert(test);</script>'];
+const validationValues = ['2147483648', '-1', '0'];
```

Comment:
Not sure again what this number means `2147483648 `. Need comments please.

Reply:
@atymchuk Just testing values: I followed this test case: https://jira.percona.com/secure/Tests.jspa#/testCase/97 :)